src/Warehouse.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+
+public class Warehouse {
+	
+	private String warehouseId;
+	private boolean receipt;
```

Comment:
I recommend using a verb here like "receiving" instead of receipt, and the companion method that checks it to be almost a question, like "isReceiving" (true/false).
```suggestion
	private boolean receiving;
```